### PR TITLE
Switch license to AGPL

### PR DESCRIPTION
**Pull request by csweichel**
## Description
This PR switches our code over to AGPL.

Thank you @meysholdt for your help with this work.

## Release Notes

```release-note
Make code available as AGPL
```

### CODE REVIEW

**Review by easyCZ — APPROVED** [review]

**Review by vulkoingim — APPROVED** [review]

**Review by utam0k — APPROVED** [review]

**Review by Pothulapati — APPROVED** [review]

File: components/ee/agent-smith/cmd/config-schema.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
-// Licensed under the Gitpod Enterprise Source Code License,
-// See License.enterprise.txt in the project root folder.
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
```
Comment (felladrin) [inline components/ee/agent-smith/cmd/config-schema.go:3]: Comment (non-blocking): In the root, there's only "License.AGPL.txt".

Comment (csweichel) [reply components/ee/agent-smith/cmd/config-schema.go]: Turns out this has always been wrong. Fixed now.

**Review by akosyakov — APPROVED** [review]

**Review by geropl — APPROVED** [review]
:heavy_check_mark:

**Review by Furisto — APPROVED** [review]

**Review by aledbf — APPROVED** [review]

**Comment by ghuntley** [comment]
Solid choice and a great outcome. I mean that.

**Pull request merged**